grid/chat/get-messages/maps/slack.suma

```diff
@@ -0,0 +1,91 @@
+profile = "chat/get-messages@1.0"
+provider = "slack"
+
+"""
+GetMessages map
+"""
+map GetMessages {
+	http GET "/conversations.history" {
+		request "application/x-www-form-urlencoded" {
+			query {
+				channel = input.destination
+				oldest = input.afterDate
+				cursor = input.page
+				limit = input.limit || 100
+			}
+
+			headers {
+				"Authorization" = `Bearer ${parameters.accessToken}`
+			}
+		}
+
+		response 200 "application/json" {
+			// See https://api.slack.com/methods/conversations.history#errors for more information about errors
+			return map error if (body.ok === false) {
+				title = body.error
+			}
+
+			messages = call MapMessages(messages = body.messages)
+
+			return map result if (body.has_more) {
```

Comment:
Looks like Slack dosn't support fetching previous page.

Reply:
Yeah, I didn't find a way to link previous pages with slack. 🤷🏻‍♂️